Reduce required env vars to just the one for datomic URI.

mgrbyte opened the pull request:
* Removed the need for WS_VERSION
 * Renamed env-var `TRACE_DB` to `WB_DB_URI`
   * Datomic URI should be specified the same way in all clj projects *
     now.
 * Updated the ElasticBeanstalk .config file accordingly.

Fixes #58

Commit:
Reduce required env vars to just the one for datomic URI.

 * Removed the need for WS_VERSION
 * Renamed env-var `TRACE_DB` to `WB_DB_URI`
   * Datomic URI should be specified the same way in all clj projects *
     now.
 * Updated the ElasticBeanstalk .config file accordingly.

Fixes #58

adamjohnwright commented on Makefile:
```diff
@@ -1,6 +1,5 @@
 NAME := wormbase/datomic-to-catalyst
 VERSION ?= $(shell git describe --abbrev=0 --tags)
-WS_VERSION := WS257
 LOWER_WS_VERSION = `echo $(WS_VERSION) | tr A-Z a-z`
```
I am not sure what this line is for, and there appears to be WS_VERSION in this file a couple more places. Any ideas?

mgrbyte replied:
I seem to have missed a thing or to in the Makefile. will fix up the PR.

Commit:
Removed usage of the old WS_VERISON env-var in Makefile.

 * The environment variable WS_VESRION is not set or passed to any
   process.
 * WS_VERSION is set in the `Makefile` only, for use in specifying the
   name of ElasticBeanstalk environment via `eb create`, and to obtain
   JSON assemblies file from FTP.

Commit:
Tidy-up Makefile syntax.

mgrbyte commented:
@a8wright I've tested:
  * make get-assemblies-json 
  * make docker-build
  * make docker-run
